### PR TITLE
completed brochure-form-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo! Your labels do not match the example in font family, indentation, or size. Additionally, the submit button is not aligned to the right side of the `select` element, as in the example. Please revise and submit a new screenshot with your revision.

**Comment by rbomercado** [comment]
image
 updated!

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo, now your header is misaligned with the form. As a reminder, please make sure you compare your output to the example side-by-side before submitting a new version.

**Comment by rbomercado** [comment]
image

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  margin: auto;
+  max-width: 800px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:7]: If you change this to something smaller (maybe like 30%), you shouldn't need to push all your content around with padding anymore.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo! One note in the code to get you started. Additionally, your header is still not centered, and the labels for each form control are meant to be in all capital letters. Please revise.

File: css-brochure-form/styles.css (hunk 1 above)
Comment (rbomercado) [reply css-brochure-form/styles.css]: no padding this time!
image

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  color: rgb(235 99 92);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  margin-right: 59px;
+}
+
+.align-center {
+  padding-left: ;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:21]: You should delete this rule, and remove the `align-center` class from any elements that have it, now that it doesn't do anything.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**